src/Raven.Studio/wwwroot/App/views/manage/cluster.html

```diff
@@ -40,7 +40,13 @@ <h3 data-bind="visible: $root.topology().leader() !== tag()">
                             <div class="address">
                                 <h5>ADDRESS</h5>
                                 <a target="_blank" data-bind="attr: { href: serverUrl }"><small data-bind="text: serverUrl"></small></a>
-                            </div>                            
+                            </div>
+                            <div data-bind="if: assignedCores">
+                                <span>Assigned Cores: </span> <span data-bind="text: assignedCores"></span>
```

Comment:
I wonder if assigned cores should be merged with total cores like:
Assigned 3 out of 8 cores ?